### DISCUSSION TITLE
Migration from S3 compatible storage to GCS

### DISCUSSION THREAD

**Discussion by marioanton** [grafana/mimir#8250] [Help and support] [answered]
Is there some pattern other than copying files over to destination and run MIMIR from there?

**Comment by pstibrany** [comment, marked as answer]
Not really, that's what we do too, when we're migrating tenants or entire Mimir installations. There's a `copyblocks` too for copying blocks between buckets/providers as well in the repository.

**Comment by marioanton** [comment]
Thank you buddy

**Comment by marioanton** [comment]
Will this work for tempo too btw?

**Reply by pstibrany** [reply]
You mean `copyblocks` tool? No, it won't work, it only understands structure of Mimir's storage and TSDB blocks.
